.devcontainer/docker-compose-devcontainer.yml

```diff
@@ -0,0 +1,194 @@
+services:
+  devcontainer:
+    image: 'mcr.microsoft.com/devcontainers/typescript-node:1-18-bullseye'
+    volumes:
+      - ..:/workspaces:cached # mounts our local directory (git root) to the container in /workspaces directory (with access mode of cached)
```

Comment:
does that really make sense considering the os/architecture may be very different?